### PR TITLE
[FLINK-6438] Added a few links to the docs home page ...

### PR DESCRIPTION
**Pull request by alpinegizmo**
… and made some other small adjustments.

### CODE REVIEW

**Comment by StephanEwen** [comment]
I like these additional links.

Since some of the links point for example to the blog of a company, would be good to get an opinion from someone else as well (who is not involved with data Artisans)

File: docs/index.md [hunk 1]
```diff
@@ -33,9 +33,13 @@ Apache Flink is an open source platform for distributed stream and batch data pr
 
 - **Concepts**: Start with the basic concepts of Flink's [Dataflow Programming Model](concepts/programming-model.html) and [Distributed Runtime Environment](concepts/runtime.html). This will help you to fully understand the other parts of the documentation, including the setup and programming guides. It is highly recommended to read these sections first.
 
-- **Quickstarts**: [Run an example program](quickstart/setup_quickstart.html) on your local machine or [write a simple program](quickstart/run_example_quickstart.html) working on live Wikipedia edits.
+- **Quickstarts**: [Run an example program](quickstart/setup_quickstart.html) on your local machine or [study some examples](examples/index.html).
 
-- **Programming Guides**: You can check out our guides about [basic concepts](dev/api_concepts.html) and the [DataStream API](dev/datastream_api.html) or [DataSet API](dev/batch/index.html) to learn how to write your first Flink programs.
+- **Programming Guides**: You can check out our guides about [basic api concepts](dev/api_concepts.html) and the [DataStream API](dev/datastream_api.html) or [DataSet API](dev/batch/index.html) to learn how to write your first Flink programs.
```
Comment (greghogan) [inline docs/index.md]: Capitalize 'API'.

File: docs/index.md [hunk 2]
```diff
@@ -33,9 +33,13 @@ Apache Flink is an open source platform for distributed stream and batch data pr
...
+
+## Deployment
+
+Before putting your Flink job into production, be sure to read the [Production Readiness Checklist](http://localhost:4000/ops/production_ready.html).
```
Comment (greghogan) [inline docs/index.md]: Fix URL.

Comment (alpinegizmo) [inline docs/index.md]: That's embarrassing; glad you caught that.

**Comment by alpinegizmo** [comment]
@greghogan I'd be happy to list other blogs; any suggestions? I've considered at a few, but the problem with the ones I've examined is that the proportion of Flink-related posts is rather small, and the posts aren't tagged, so we can't link directly the subset that's relevant.

**Comment by greghogan** [comment]
@alpinegizmo understood, and meant to say that with an eye to the future. Was only confirming that the community is not giving special consideration to data Artisians.

**Pull request closed without merging**